<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parametersDW</class>
 <widget class="QDockWidget" name="parametersDW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>642</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelBiotWillis">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Biot-Willis a:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alpha">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelPermeability">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Permeability k=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="k">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Material:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="materialCBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Sandler e Dimaggio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mohr-Coulomb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear el√°stico</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="sandlerGrp">
      <property name="title">
       <string>Sandler</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>E:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLineEdit" name="fB"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="fW"/>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>D:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="fD"/>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Poisson:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>C:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>W:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="fR"/>
       </item>
       <item row="12" column="0">
        <widget class="QLineEdit" name="fC"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>A:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLineEdit" name="fA"/>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="fE"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="fPoisson"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="mohrCGrp">
      <property name="title">
       <string>Mohr</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cohesion:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Angle:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="fCohesionMohr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fAngleMohr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Young:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Poisson:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="fYoungMohr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fPoissonMohr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="elasticGrp">
      <property name="title">
       <string>Linear elastic</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Young:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Poisson:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="fYoungLinear"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fPoissonLinear"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>288</width>
        <height>4</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
